pandas/tseries/index.py

```diff
@@ -814,8 +815,7 @@ def _add_offset(self, offset):
                           "or DatetimeIndex", PerformanceWarning)
             return self.astype('O') + offset
 
-    def _format_native_types(self, na_rep=u('NaT'),
-                             date_format=None, **kwargs):
+    def _format_native_types(self, na_rep='NaT', date_format=None, **kwargs):
```

Comment:
Yes. The test is among astype tests for DatetimeIndex (just an `assert_index_equal()`, which previously discovered the Indexes had different inferred types). 
I'll comment. Or should I rather make it more explicit and place somewhere else?